logic/vote.js

```diff
@@ -95,6 +95,12 @@ Vote.prototype.verify = function (trs, sender, cb) {
 		return setImmediate(cb, 'Voting limit exceeded. Maximum is 33 votes per transaction');
 	}
 
+	for (var i = 0; i < trs.asset.votes.length; i++) {
+		if (trs.asset.votes.filter(function (v) { return v === trs.asset.votes[i]; }).length > 1) {
+			return setImmediate(cb, 'Duplicate votes are not allowed');
+		}
+	}
```

Comment:
Didn't want to import lodash in the file for use in only one condition. But non-nested if condition looks better. So, I'll update it. :+1: Thanks